README.rst

```diff
@@ -14,9 +14,9 @@ It is
   and provides a simple, high-level API, compatible with xarray_ as well as basic numpy arrays.
 - **Fast**: It is faster than ESMPy's original Fortran regridding engine in serial case, and also supports dask_ for `out-of-core, parallel computation <http://xarray.pydata.org/en/stable/dask.html>`_.
 
-Please see `online documentation <http://xesmf.readthedocs.io/en/latest/>`_, or `play with example notebooks on Binder <https://mybinder.org/v2/gh/pangeo-data/xESMF/master?filepath=doc%2Fnotebooks>`_.
+Please see `online documentation <http://pangeo-xesmf.readthedocs.io/en/latest/>`_, or `play with example notebooks on Binder <https://mybinder.org/v2/gh/pangeo-data/xESMF/master?filepath=doc%2Fnotebooks>`_.
```

Comment:
I am curious... Are we doing this because we are unable to get hold of Jiawei for the time being? What will happen to the future releases? Will we need to publish `xesmf` under new name `pangeo-xesmf` on PyPI and conda-forge?

Reply:
yes, right now the link is to the old docs and the badge shows docs building but they're not. I propose we fix this by pointing to the pangeo-xesmf docs. We can always revert when we get the official xESMF docs migrated